game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -22,6 +22,13 @@ object Cell {
    * Ratio of mass lost per second.
    */
   final val MassDecayPerSecond = 0.005f
+
+  /**
+    * Ratio of impact of the mass on the acceleration.
+    */
+  final val MassImpactOnAcceleration = 0.75f
+
+  final val MinSpeed = 25f
```

Comment:
Could we rename this? I really thought you were adding a minimum speed for *any* movement at first (not on the max speed).
Suggestion: `MinSpeedLimit`